optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -610,3 +670,138 @@ def _reshape(self, model: openvino.runtime.Model, batch_size: int, sequence_leng
                     shapes[inputs][1] = -1
         model.reshape(shapes)
         return model
+
+
+@add_start_docstrings(
+    """
+    Speech Sequence-to-sequence model with a language modeling head for OpenVINO inference. This class officially supports whisper, speech_to_text.
+    """,
+    INPUTS_DOCSTRING,
+)
+class OVModelForSpeechSeq2Seq(OVModelForSeq2SeqLM):
+    auto_model_class = AutoModelForSpeechSeq2Seq
+    main_input_name = "input_features"
+    export_feature = "automatic-speech-recognition"
+
+    def prepare_inputs_for_generation(
+        self,
+        input_ids,
+        input_features: Optional[torch.FloatTensor] = None,
```

Comment:
Is this needed ?
```suggestion

```
I don't see it  in
https://github.com/huggingface/transformers/blob/main/src/transformers/models/whisper/modeling_whisper.py#L2494
or 
https://github.com/huggingface/transformers/blob/a49f4acab3c1eea82907e12f82eafbd4673deb39/src/transformers/models/speech_to_text/modeling_speech_to_text.py#L1339

Reply:
removed